android/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+SET(Libs_DIR ${Libs_DIR} ${CMAKE_SOURCE_DIR}/android/app/libs)
+SET(Include_DIR ${Libs_DIR}/include)
+SET(Library_DIR ${Libs_DIR}/lib/${CMAKE_ANDROID_ARCH_ABI})
+include_directories(${Include_DIR})
+
+SET(LUAJIT_INCLUDE_DIR ${LUAJIT_INCLUDE_DIR} ${Include_DIR}/luajit)
```

Comment:
The actual vcpkg port for Android doesn't compile, their build is broken for Android.